server/src/main/java/io/druid/segment/realtime/appenderator/Appenderators.java

```diff
@@ -129,5 +128,4 @@ public boolean isAnnounced(DataSegment segment)
     );
   }
 
-
```

Comment:
this is part of bug fix. when using offline appenderator we don't server queries so no need for cache.